en/docs/learn/account-locking-by-failed-login-attempts.md

```diff
@@ -139,20 +186,8 @@ tenants.
 
 !!! note "Configuring WSO2 IS for automatic account unlock"
       
-    The WSO2 Identity Server can be configured to automatically unlock a
-    user account after a certain period of time. A user account locked by
-    failed login attempts can be unlocked by setting a lock timeout period.
-    
-    Configure the `         Authentication.Policy.Account.Lock.Time        `
-    property in the `        `
-    `         <IS_HOME>/repository/conf/identity/identity-mgt.properties        `
-    file . As mentioned in the above table, the value refers to the number
-    of minutes that the account is locked for, after which, authentication
-    can be attempted again.
-    
-    ``` bash
-    Authentication.Policy.Account.Lock.Time=5 
-    ```
+    A user account locked by failed login attempts can be unlocked 
+    by setting the <strong>Account Unlock Time</strong> period.
```

Comment:
shall we add a sample code block with this property?

Reply:
It's available in earlier in the page.